terraform/modules/erouska/efgs.tf

```diff
@@ -0,0 +1,52 @@
+locals {
+  uploadkeystoefgs_invoker_roles = [
+    "roles/cloudfunctions.serviceAgent",
+    "roles/iam.serviceAccountUser"
+  ]
+
+  uploadkeystoefgs_roles = [
```

Comment:
I believe you're missing a service account and IAM member definition for `UploadKeysToEFGS` - these roles are used nowhere.